content/books/golden-age-of-intelligence/chapter-02-the-twenty-watt-miracle.md

```
@@ -0,0 +1,202 @@
+# Chapter 2 — The 20-Watt Miracle
+
+> *"The All is Mind. The universe is mental."*
+> — Hermetic principle, *The Kybalion* (Three Initiates, 1908, summarizing the seven Hermetic principles)
+
```

markdownlint-cli2 (0.22.1)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)